Algorithm.CSharp/Alphas/VixDualThrustAlphaAlgorithm.cs

```diff
@@ -0,0 +1,265 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using QuantConnect.Algorithm.Framework;
+using QuantConnect.Algorithm.Framework.Alphas;
+using QuantConnect.Algorithm.Framework.Execution;
+using QuantConnect.Algorithm.Framework.Portfolio;
+using QuantConnect.Algorithm.Framework.Risk;
+using QuantConnect.Algorithm.Framework.Selection;
+using QuantConnect.Brokerages;
+using QuantConnect.Data;
+using QuantConnect.Data.Consolidators;
+using QuantConnect.Data.Market;
+using QuantConnect.Data.UniverseSelection;
+using QuantConnect.Indicators;
+
+namespace QuantConnect.Algorithm.CSharp.Alphas
+{
+    class VIXDualThrustAlphaAlgorithm : QCAlgorithmFramework
```

Comment:
Please rename to end in Alpha, pedantic but keeps the pattern of the other alphas